bundles/org.openhab.binding.konnected/src/main/resources/OH-INF/i18n/konnected.properties

```diff
@@ -19,6 +19,8 @@ thing-type.config.konnected.module.controller_removewifi.label = Factory Reset
 thing-type.config.konnected.module.controller_removewifi.description = Resets the module to Factory Conditions.
 thing-type.config.konnected.module.controller_sendConfig.label = Update Settings
 thing-type.config.konnected.module.controller_sendConfig.description = Manually sends the settings to the module. The binding will send settings on every restart and if there are any configuration changes but this can be used to manually update the settings as needed.
+thing-type.config.konnected.module.callback_uri.label = Callback URI
```

Comment:
Please regenerate this file as it should contain more of the changes provided in thing-types.xml. See https://next.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file.

Reply:
Done

Reply:
Is must be regenerated again after the latest changes.

Reply:
Done